main.c

```diff
@@ -70,16 +65,12 @@ gmp_free_function(void *ptr, size_t size)
     free(ptr);
 }
 
-
-
 typedef struct _mpzobject {
-    PyObject_HEAD
-    uint8_t negative;
+    PyObject_HEAD uint8_t negative;
```

Comment:
?!

Reply:
Since it is a macro, it seems to be assumed by default that there is no trailing whitespace at the end. Hmm… This issue is quite serious.